Makefile

```diff
@@ -145,3 +145,10 @@ test-parallel: build-in-docker
 lab: build-musl
 	cp -v .musl-build/ngx_http_datadog_module.so* lab/services/nginx/
 	lab/bin/run $(TEST_ARGS)
+
+.PHONY: circleci-config
+circleci-config:
+	@echo "Compiling circleci config"
+	circleci config pack .circleci/src > .circleci/continue_config.yml
```

Comment:
This command will not include the content of @common.yml and @jobs.myl files. Is this intended ?

Reply:
Ok, that is just that it is all over the place in the final file, but it does not really matter